docs/contributing-to-airbyte/building-new-connector/legacy-standard-source-tests.md

```diff
@@ -2,7 +2,7 @@
 
 ## Airbyte's Standard Tests (v1)
```

Comment:
should we also move this page to the new connector dev section?

Reply:
Done.